src/Anonymization/Anonymizator.php

```diff
@@ -52,12 +58,17 @@ protected function createAnonymizer(AnonymizerConfig $config): AbstractAnonymize
         $className = $this->anonymizerRegistry->get($config->anonymizer);
         \assert(\is_subclass_of($className, AbstractAnonymizer::class));
 
-        return new $className(
+        $ret = new $className(
             $config->table,
             $config->targetName,
             $this->connection,
             $config->options,
         );
+        \assert($ret instanceof AbstractAnonymizer);
+
+        $ret->setSalt($this->getSalt());
+
+        return $ret;
```

Comment:
Every `Anonymizer` has to implement `AbstractAnonymizer`, so we could just do

```suggestion
        return new $className(
            $config->table,
            $config->targetName,
            $this->connection,
            $config->options,
            $this->getSalt(),
        );
        return $ret;
```

no ?

Reply:
Yes, indeed, I'll change that.